.github/workflows/ci.yml

```diff
@@ -235,7 +235,32 @@ jobs:
           path: |
             kotlin/hello.exe
             kotlin/info.txt
-
+            
+  build_nim:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: jiro4989/setup-nim-action@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build
+        shell: cmd
+        run: |
+          cd nim
+          nim c --opt:size -d:release hello.nim
```

Comment:
`-d:release` - specifies that we need a release build, like `dotnet build -c Release`
The "cheating" part may be --opt:size (optimize for size), that could be removed probably

Reply:
Release build won't enable optimizations?

Reply:
It will, but I believe all of the builds in the repo are release builds ?
Otherwise what's the reason to compare debug builds
For dotnet 8 for example, they've done dotnet pack / dotnet publish enable Release by default

Reply:
Yup, `dotnet build -c Release` will enable optimizations for C#, so we don't need to pass `dotnet build -c Release -p:Optimize=true` because Release will set that.

I'm just wondering whether nim command line really needs to say "build release configuration" and then "no, really, give me the bits I can release" again or whether skipping `--opt` will still optimize.

Reply:
The let's remove that part - it's the "You can enable optimizations if optimizations are not enabled by default. If the compiler has a master switch "enable optimizations", that switch should be used. If there are only more specialized switches (like in Zig), we have no option but to optimize for size, although the results are less representative of what is canonical." rule. The master switch is `-d:release` then.

Reply:
Done